custom/aaa/tests/test_utils.py

```diff
@@ -333,4 +337,7 @@ def test_build_location_filters_for_state_without_child_MWCD(self):
     def test_build_location_filters_for_national_without_child_MWCD(self):
         expected_output = {}
         result = build_location_filters('', MINISTRY_MWCD, with_child=False)
-        self.assertDictEqual(result, expected_output)
\ No newline at end of file
+        self.assertDictEqual(result, expected_output)
+
+    def slow_test(self):
+        time.sleep(5)
```

Comment:
Is this intended to be part of the final state of this PR? Or maybe you were just verifying that the timing plugin works the way you expected and were planning to remove it?

Reply:
Correct. This wouldnt be in the final PR. just using it for an example testing